packages/launcher/src/main/rpc/app.js

```diff
@@ -61,6 +62,67 @@ export const sandboxed = {
     return accounts
   },
 
+  // Contacts
+
+  contacts_select: withPermission(
+    'CONTACTS_READ',
+    async (ctx: AppContext, params: { multi?: boolean }) => {
+      const res = await ctx.trustedRPC.request('permission_ask', {
+        key: 'CONTACTS_SELECT_CONTACTS',
+        params,
+      })
+      if (!res.granted || !res || !res.data) {
+        return { contacts: [] }
+      }
+      const userID = ctx.appSession.user.id
+      const appID = ctx.appSession.app.appID
+      const contactIDs = res.data
+      const contactsToApprove = contactIDs.map(id => ({
+        localID: id,
+        publicDataOnly: false, // TODO allow user to set only public data
```

Comment:
Could we default this to `true`, so if we add support for private data later we can let the user opt-in?